dpnp/dparray.pyx

```diff
@@ -281,6 +372,26 @@ cdef class dparray:
 
         return interface_dict
 
+
+    @property
+    def __sycl_usm_array_interface__(self):
+        cdef dict iface = {
```

Comment:
why you use Cython specific things in pure Python procedure? It will requires extra data conversion.

Reply:
I removed it.